integration-tests/main/src/test/java/io/quarkus/it/main/OpenApiTestCase.java

```diff
@@ -22,7 +22,7 @@
 
     private static final String DEFAULT_MEDIA_TYPE = "application/json";
 
-    @TestHTTPResource("openapi")
+    @TestHTTPResource("q/openapi")
```

Comment:
This is one of the only places w/o the leading '/' ... 
while it may work, the inconsistency is noticable